sci-astronomy/stellarium/files/stellarium-0.23.3-ccache.patch

```diff
@@ -0,0 +1,13 @@
+Don't automagically use ccache https://bugs.gentoo.org/847439
+
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -196,7 +196,7 @@ IF(OPENGL_DEBUG_LOGGING)
+ ENDIF()
+ 
+ # Use ccache if possible
+-IF(NOT WIN32)
++IF(FALSE)

```

Comment:
can we convince upstream to turn this into an option? ccache automagic in upstream buildsystem is baaad.

Reply:
*Kinda* is already is an option given find_program is easy to override, still not a great way for upstream to handle this though.

Haven't tried but `-DCCACHE_PROGRAM=no` probably works.

Reply:
Actually just tried, and can confirm it does work -- so no patch needed.